dart-impl/mpi/src/dart_team_group.c

```diff
@@ -27,9 +27,10 @@
 
 static struct dart_group_struct* allocate_group()
 {
-  struct dart_group_struct* group = malloc(sizeof(struct dart_group_struct));
+  struct dart_group_struct* group = (struct dart_group_struct *) malloc(
```

Comment:
Cast of `malloc` return type is not needed in ANSI C, see http://c-faq.com/malloc/cast.html